cmake/GlowConvertCL.cmake

```diff
@@ -0,0 +1,3 @@
+include(${CMAKE_CURRENT_LIST_DIR}/modules/BinToHeader.cmake)
+message(STATUS "Generating: ${HEADER_PATH} from ${BIN_PATH}")
+BinToHeader(SOURCE_FILE "${BIN_PATH}" VARIABLE_NAME ${VARIABLE_NAME} HEADER_FILE "${HEADER_PATH}" NULL_TERMINATE)
```

Comment:
I'm not sure NULL_TERMINATE works as expected. When I look at the generated files, I see that the last byte is e.g. 0xa and not 0x0. Could you check?

Reply:
Yeap, you are right, fixed.